lib/rules/selector-pseudo-element-allowed-list/index.js

```diff
@@ -47,7 +47,7 @@ const rule = (primary) => {
 					const value = pseudoNode.value;
 
 					// Ignore pseudo-classes
-					if (value[1] !== ':') {
+					if (value.charAt(1) !== ':') {
```

Comment:
[note] I think the [`charAt()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/charAt) method is more readable since we can easily see that its method receiver is a string.